src/locale/locales/es/messages.po

```diff
@@ -3755,64 +3755,64 @@ msgstr ""
 
 #: src/view/com/notifications/FeedItem.tsx:211
 msgid "liked your custom feed"
-msgstr ""
+msgstr "le gusta tu feed personalizado"
 
 #: src/view/com/notifications/FeedItem.tsx:178
 msgid "liked your post"
-msgstr ""
+msgstr "le gusta tu post"
 
 #: src/view/screens/Profile.tsx:222
 msgid "Likes"
-msgstr "Cantidad de «Me gusta»"
+msgstr "Me gusta"
 
 #: src/view/com/post-thread/PostThreadItem.tsx:204
 msgid "Likes on this post"
-msgstr ""
+msgstr "Me gusta en este post"
 
 #: src/Navigation.tsx:192
 msgid "List"
-msgstr ""
+msgstr "Lista"
 
 #: src/view/com/modals/CreateOrEditList.tsx:241
 msgid "List Avatar"
 msgstr "Avatar de la lista"
 
 #: src/view/screens/ProfileList.tsx:413
 msgid "List blocked"
-msgstr ""
+msgstr "Lista de bloqueados"
 
 #: src/components/ListCard.tsx:150
 #: src/view/com/feeds/FeedSourceCard.tsx:252
 msgid "List by {0}"
-msgstr ""
+msgstr "Lista por {0}"
 
 #: src/view/screens/ProfileList.tsx:450
 msgid "List deleted"
-msgstr ""
+msgstr "Lista de eliminados"
 
 #: src/screens/List/ListHiddenScreen.tsx:126
 msgid "List has been hidden"
-msgstr ""
+msgstr "Se ha ocultado la lista"
 
 #: src/view/screens/ProfileList.tsx:161
 msgid "List Hidden"
-msgstr ""
+msgstr "Lista oculta"
 
 #: src/view/screens/ProfileList.tsx:387
 msgid "List muted"
-msgstr ""
+msgstr "Lista silenciada"
 
 #: src/view/com/modals/CreateOrEditList.tsx:255
 msgid "List Name"
 msgstr "Nombre de la lista"
 
 #: src/view/screens/ProfileList.tsx:426
 msgid "List unblocked"
-msgstr ""
+msgstr "Lista desbloqueada"
 
 #: src/view/screens/ProfileList.tsx:400
 msgid "List unmuted"
-msgstr ""
+msgstr "Lista desilenciada"
```

Comment:
`Lista no silenciada`. Same like the other "unmuted", I'd just try to keep it consistent.

Reply:
These are tricky, because this string is an action confirmation after you unmute a list. It's not that "the list is not muted", it's that "the list is no longer muted", it should indicate a change in status. But again, "desilenciado/a" sounds a little off.

Reply:
Would something like `Lista ya no está silenciada` work?

Reply:
That should be "La lista ya no está silenciada". I also suggested elsewhere "Ya no estás silenciando esta lista"

Reply:
Fixed as voted